stable/aws-for-fluent-bit/values.yaml

```diff
@@ -102,8 +102,12 @@ resources:
   limits:
     memory: 500Mi
   requests:
-    cpu: 500m
-    memory: 500Mi
+    cpu: 50m
+    memory: 50Mi
```

Comment:
Please check [this comment](https://github.com/aws/eks-charts/pull/104#discussion_r406160004) in the initial PR for the reason of the existing limits
cc: @max-rocket-internet

Reply:
I might be wrong but i'm not sure setting limit=request prevents oomkilled anyway. And since fluent bit is supposed to be resources efficient more than fluentd I think it is a bit excessive to "waste" 500Mi of RAM on a node which cannot be used for other pods in that case the gains compared to fluentd are lost.

But we can leave it this way and let the users change the default values anyway.